.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+	"trailingComma": "none",
+	"bracketSpacing": false,
+	"arrowParens": "avoid",
+	"singleQuote": false,
+	"printWidth": 80,
+	"useTabs": false,
```

Comment:
I don't want to start that debate lol but can we use tabs here?

Reply:
We discussed that previously and came to this https://github.com/saadpasta/developerFolio/pull/107#discussion_r432846616

Reply:
Oh, I didn't remember that. Sure, this is alright.